src/composables/forms.ts

```diff
@@ -63,19 +67,21 @@ export const defaultInputProps = {
   placeholder: "",
 }
 
-export type TextInputType =
-  | "date"
-  | "datetime-local"
-  | "email"
-  | "month"
-  | "number"
-  | "password"
-  | "search"
-  | "tel"
-  | "text"
-  | "time"
-  | "url"
-  | "week"
+export const textInputTypes = [
+  "date",
+  "email",
+  "month",
+  "number",
+  "password",
+  "search",
+  "tel",
+  "text",
+  "time",
+  "url",
+  "week",
+] as const
+
+export type TextInputType = (typeof textInputTypes)[number]
```

Comment:
This will allow me to remove a small copy/paste in the `FieldsSchema` work on portal, see [#1220](https://github.com/xy-planning-network/college-try/pull/1220)